package.json

```diff
@@ -8,8 +8,10 @@
     "check-types": "tsc",
     "preview": "vite preview",
     "lint": "eslint --max-warnings 0 'src/**/*.ts' 'src/**/*.tsx' && knip",
+    "lighthouse": "SCRIVITO_ORIGIN=http://localhost:8080 PRERENDER_OBJ_ID=c2a0aab78be05a4e npm run prerender && npm run prerender-precompress && for run in $(seq 6); do lhci autorun && break; done",
     "prerender": "vite build && vite-node --options.deps.external='/scrivito/' src/prerenderContent.ts",
-    "prerender-preview": "vite preview --outDir dist-ssr",
+    "prerender-precompress": "precompress --include '(**).(css|html|js)' --types br dist-ssr",
+    "prerender-preview": "ws --directory dist-ssr --port 8080 --rewrite '(/..)(/.*|$) -> $&.html' --static.maxage 31536000",
```

Comment:
We are getting closer. Now only a fallback file is missing:

*How to reproduce*
```
SCRIVITO_ORIGIN=http://localhost:8080 npm run prerender && npm run prerender-preview
```
Visit http://localhost:8080/en/my-tynacoon

*Expected*
A nicely rendered "Page not found"
<img width="1624" alt="Screenshot 2024-08-08 at 09 30 01" src="https://github.com/user-attachments/assets/024af2b0-4018-4642-94ad-ba6711456e55">

*Actual*
A white page stating "Not found"
<img width="1624" alt="Screenshot 2024-08-08 at 09 30 31" src="https://github.com/user-attachments/assets/b9dfee68-1fcb-4725-b428-2c3655ba3352">